src/main/com/yetanalytics/datasim/input/alignments.clj

```diff
@@ -17,7 +40,8 @@
 
 (s/def ::alignment
   (s/keys :req-un [::component
-                   ::weight]))
+                   ::weight]
+          :opt-un [::objectOverride]))
 
 ;; Actor-Alignment: Map of Actor to collection of Alignments
```

Comment:
I'm guessing @cliffcaseyyet didn't delete this comment when he overhauled alignments